app/phpunit.xml.dist

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file is added, why so? (if it can't be thought it, it's likely it's not needed and should be added to the `.gitignore` file.